modules/apps/adaptive-media/adaptive-media-image-impl/src/main/java/com/liferay/adaptive/media/image/internal/scaler/AMImageMagickImageScaler.java

```diff
@@ -43,12 +47,10 @@
 
 /**
  * @author Eric Yan
+ * @author Roberto Díaz
  */
 @Component(
-	property = {
-		"mime.type=image/gif", "mime.type=image/heic", "mime.type=image/tiff",
-		"mime.type=image/webp"
-	},
+	property = {"mime.type=*", "service.ranking:Integer=101"},
```

Comment:
The default service ranking is 0; for consistency it is better to use the value 100 as in other places.